lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -68,8 +68,6 @@ export class SearchFacetFiltersService {
             this.responseFacets = null;
         });
 
-        this.queryBuilder.updated.pipe(takeUntilDestroyed()).subscribe((query) => this.queryBuilder.execute(true, query));
```

Comment:
With this change when you change the state of any filter and click apply it won't trigger any search request